### ISSUE TITLE
Poor behavior when hovering over link text

### ISSUE THREAD

**Issue by markcmiller86** [orderedlist/minimal#56]
Consider [this page](https://visit-dav.github.io/largedata/help/adding-download-files.html) and look at the link with text in the first paragraph of the page with `GitHub’s Large File Support`. Hover over the `GitHub's` text at the end of the second to last line of the paragraph.

Without hovering, this text has a line break after `GitHub's`. With hovering, the bold font the theme creates when hovering causes the link text to move down a whole line out from under the pointer entirely. 

In safari, this creates a quick oscillation of the text (normal->bold->normal) and then the pointer is unable to click. In Chrome, the text goes bold and expands to the next line and the pointer indicates it is still clickable but clicking has no effect.

**Comment by contactrika** [comment]
Agreed, this was a very problematic issue! I think I found a fix for this:
https://github.com/orderedlist/minimal/issues/44#issuecomment-826057962

**Comment by tripleee** [comment]
The fix in the previous comment solved it for me. Somewhat to my consternation, I don't see this behavior in the demo version of this theme, but the pages built by Github Pages using this team definitely still suffer from it.
